### PR TITLE
Feat/tickets CRUD

### LINKED ISSUE #9: Tickets CRUD

### PR DESCRIPTION
**Pull request by VerasThiago**
Closes #9

### CODE REVIEW

File: api/pkg/handlers/ticket/create.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	var err error
+	var request *ticket.CreateRequest
+
+	if err = context.ShouldBindJSON(&request); err != nil {
+		return err
+	}
+
+	if errList := request.ValidateSyntax(); len(errList) > 0 {
+		return errors.CreateGenericErrorFromValidateError(errList)
+
```
Comment (diogoftp) [inline api/pkg/handlers/ticket/create.go:36]: Empty line

File: api/pkg/validator/ticket/delete.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	options := govalidator.Options{
+		Data:  d,
+		Rules: rules,
+	}
+
+	values := govalidator.New(options).ValidateStruct()
+	return validator.MergeUrlValues(validator.GetRulesKey(rules), values)
+}
+
+func (d *DeleteRequest) ValidateSemantic(user *models.User) []string {
```
Comment (diogoftp) [inline api/pkg/validator/ticket/delete.go:29]: Unused parameter

File: api/pkg/handlers/ticket/validate.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	var err error
+	var request *ticket.ValidateRequest
+
+	if err = context.ShouldBindJSON(&request); err != nil {
+		return err
+	}
+
+	if errList := request.ValidateSyntax(); len(errList) > 0 {
+		return errors.CreateGenericErrorFromValidateError(errList)
+
```
Comment (diogoftp) [inline api/pkg/handlers/ticket/validate.go:36]: Empty line

**Review by diogoftp — APPROVED** [review]

**Review by murillomuller — APPROVED** [review]
LGTM

**Pull request merged**